✨ Add one-click attacker cluster blocking documentation (#37719)
DataDog/documentation: content/en/security/application_security/security_signals/attacker_clustering.md

{{< img src="security/application_security/threats/custom-clusters.png" alt="An AAP signal with custom clusters sorted by the attacker attributes"  >}}

> ## Block an attacker cluster in one click

Reviewer: ```suggestion
## Block an attacker cluster with one click
```

> After an attacker cluster is identified, you can generate an In-App WAF custom rule that matches its attributes directly from the UI without writing regex.

Reviewer: For "matches its attributes", is it the attributes of the attacker cluster?

```suggestion
After an attacker cluster is identified, you can directly generate an In-App WAF custom rule in the UI that matches its attributes, without needing to write regex.
```

Author: Yes, the attributes are the cluster attributes. I've clarified that part.

To block a cluster:

1. Open the **Attacker Explorer** and select the **Cluster** grouping.
2. Click on a cluster to open its side panel.
3. Click **Create In-App WAF rule** in the cluster header. The In-App WAF custom rule form opens pre-filled with the generated conditions, one condition per blocking attribute, combined with AND logic.
4. Review the conditions and adjust if needed, then save the rule.

> You can also click the **Create In-App WAF rule** icon from a cluster row in the clusters table inside a security signal.

Reviewer: ```suggestion
Alternatively, inside a security signal, you can click the **Create In-App WAF rule** icon on the cluster row in the clusters table.
```
